pkg/indicator/v2/momentum/stoch.go

```diff
@@ -26,15 +27,15 @@ type StochStream struct {
 	window  int
 	dPeriod int
 
-	highPrices, lowPrices *PriceStream
+	highPrices, lowPrices *v2.PriceStream
 
 	updateCallbacks []func(k, d float64)
 }
 
 // Stochastic Oscillator
-func Stoch(source KLineSubscription, window, dPeriod int) *StochStream {
-	highPrices := HighPrices(source)
-	lowPrices := LowPrices(source)
+func Stoch(source v2.KLineSubscription, window, dPeriod int) *StochStream {
```

Comment:
you can use "GoLand"'s refactor tool to move the package,

we should keep all indicators in the v2 package to make it simple, they will reference each other.